bindings/matrix-sdk-crypto-ffi/src/olm.udl

```diff
@@ -1,4 +1,4 @@
-namespace olm {
+namespace matrix_crypto_ffi {
```

Comment:
this looks like it could have a bad down-the-line impact ... are we sure this isn't required somewhere? (I remember the Android team or some other people insisting we keep the `olm` name here for some reason...)

Reply:
I don't know personally. cc @poljar

Reply:
I don't remember people insisting, though they might prefer it. :shrug: